tests/e2e/dsc_cfmap_deletion_test.go

```diff
@@ -46,6 +46,11 @@ func cfgMapDeletionTestSuite(t *testing.T) {
 			require.NoError(t, err, "Error while deleting owned namespaces")
 		})
 
+		// t.Run("DS Projects should be deleted", func(t *testing.T) {
+		// 	err = testCtx.testDSProjectDeletion()
+		// 	require.NoError(t, err, "Error while deleting DS Projectss")
+		// })
```

Comment:
Why do we need to keep it in the comment?

Reply:
forgot to clean up this one